coreclient/src/clients/safety_code.rs

```diff
@@ -0,0 +1,79 @@
+// SPDX-FileCopyrightText: 2025 Phoenix R&D GmbH <hello@phnx.im>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use aircommon::{credentials::ClientCredential, identifiers::UserId};
+use anyhow::Context;
+use sha2::Digest;
+use tls_codec::{Serialize as _, TlsSerialize, TlsSize};
+
+use crate::{clients::CoreUser, groups::client_auth_info::StorableClientCredential};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, TlsSize, TlsSerialize)]
+#[repr(u8)]
+enum SafetyCodeVersion {
+    V1,
```

Comment:
I think we should have an explicit value here. I suppose `0` is implied in this case, but it's better to make it explicit and have `V1 = 1`.